File: 백준/Bronze/31798. 단원평가/단원평가.c
#include <stdio.h>
#include <string.h>
#include <math.h>

int main(void){
  int a, b, c;
  scanf("%d %d %d", &a, &b, &c);
  if (a == 0){
    printf("%d\n", (int) pow(c, 2)-b);
  }
  else if (b == 0){
    printf("%d\n", (int) pow(c, 2)-a);
  }
  else {
    printf("%d\n", (int) sqrt(a+b));
  }

  return 0;
}

File: 백준/Bronze/31798. 단원평가/단원평가.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();

        if (a == 0){
            System.out.println((int)Math.pow(c, 2)-b);
        }
        else if (b == 0) {
            System.out.println((int)Math.pow(c,2)-a);
        }
        else {
            System.out.println((int)Math.sqrt(a+b));
        }
    }

}